packages/lodestar/test/unit/executionEngine/http.test.ts

```diff
@@ -18,72 +21,90 @@ describe("ExecutionEngine / http", () => {
   let executionEngine: ExecutionEngineHttp;
   let returnValue: unknown = {};
   let reqJsonRpcPayload: unknown = {};
+  let baseUrl: string;
+  let errorResponsesBeforeSuccess = 0;
+  let controller: AbortController;
 
   before("Prepare server", async () => {
-    const controller = new AbortController();
+    controller = new AbortController();
     const server = fastify({logger: false});
 
     server.post("/", async (req) => {
-      reqJsonRpcPayload = req.body;
-      delete (reqJsonRpcPayload as {id?: number}).id;
-      return returnValue;
+      if (errorResponsesBeforeSuccess === 0) {
+        reqJsonRpcPayload = req.body;
+        delete (reqJsonRpcPayload as {id?: number}).id;
+        return returnValue;
+      } else {
+        --errorResponsesBeforeSuccess;
+        throw Error(`Will succeed after ${errorResponsesBeforeSuccess} more attempts`);
```

Comment:
if we have fetchWithRetries as part of jsonrpc class, can we add retries test as separate file? just to test retry logic and not mix it up with the execution specs.